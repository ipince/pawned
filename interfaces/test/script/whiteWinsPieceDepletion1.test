StartNewGame human 0 human 0
MakeMove b1-a3 0
MakeMove b7-b5 0
MakeMove a3-b5 0
MakeMove a7-a5 0
MakeMove b5-c7 0
MakeMove d8-c7 0
MakeMove b2-b3 0
MakeMove c7-c2 0
MakeMove d1-c2 0
MakeMove c8-b7 0
MakeMove c2-h7 0
MakeMove b7-g2 0
MakeMove h7-h8 0
MakeMove g2-f1 0
MakeMove h8-g8 0
MakeMove f1-e2 0
MakeMove e1-e2 0
MakeMove d7-d6 0
MakeMove g8-g7 0
MakeMove f8-g7 0
MakeMove e2-d1 0
MakeMove g7-a1 0
MakeMove d2-d4 0
MakeMove a1-d4 0
MakeMove g1-f3 0
MakeMove d4-f2 0
MakeMove b3-b4 0
MakeMove a5-b4 0
MakeMove h2-h4 0
MakeMove f2-h4 0
MakeMove f3-h4 0
MakeMove a8-a2 0
MakeMove h4-g2 0
MakeMove a2-g2 0
MakeMove h1-g1 0
MakeMove g2-g1 0
MakeMove d1-c2 0
MakeMove g1-c1 0