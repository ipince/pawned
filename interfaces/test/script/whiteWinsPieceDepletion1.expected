New game started
b1-a3
b7-b5
a3-b5
a7-a5
b5-c7
d8-c7
b2-b3
c7-c2
d1-c2
c8-b7
c2-h7
b7-g2
h7-h8
g2-f1
h8-g8
f1-e2
e1-e2
d7-d6
g8-g7
f8-g7
e2-d1
g7-a1
d2-d4
a1-d4
g1-f3
d4-f2
b3-b4
a5-b4
h2-h4
f2-h4
f3-h4
a8-a2
h4-g2
a2-g2
h1-g1
g2-g1
d1-c2
g1-c1
White Player has won
